providers/summary/clearlydefined.js

```diff
@@ -244,10 +244,26 @@ class ClearlyDescribedSummarizer {
         if (bugs.startsWith('http')) setIfValue(result, 'described.issueTracker', bugs)
       } else setIfValue(result, 'described.issueTracker', bugs.url || bugs.email)
     }
-    const license =
-      manifest.license &&
-      SPDX.normalize(typeof manifest.license === 'string' ? manifest.license : manifest.license.type)
-    setIfValue(result, 'licensed.declared', license)
+    // We could have singular licenses such as 'MIT' or licenses in an array ['MIT', 'BSD']
+    // Process licenses depending on whether they are strings or array of strings
+    var expression = null  // license expression (as input to SPDX.normalize)
+    if (typeof manifest.license === 'string') {
+      expression = manifest.license
+    } else if (Array.isArray(manifest.license)) {
+      expression = manifest.license.join(' OR ')
+    } else if (manifest.license && (typeof manifest.license.type === 'string')) {
+      expression = manifest.license.type // handle sub-property 'type'
+    } else if (manifest.license && (Array.isArray(manifest.license.type))) {
+      expression = manifest.license.type.join(' OR ')
+    } else if (typeof manifest.licenses === 'string') {
+      expression = manifest.licenses // handle legacy NPM 'licenses' key
+    } else if (Array.isArray(manifest.licenses)) {
+      expression = manifest.licenses.join(' OR ')
```

Comment:
This is unclear. I've commented in the ticket to get clarification. https://github.com/clearlydefined/service/issues/458

Reply:
Can you please explain what is unclear?
The idea here is the license may be expressed in the manifest under the key/property `license`,`licence.type`, or `licenses` with value datatype of array of strings (presumed to be a disjunction) or just a string.

Reply:
The code is clear. Please see my comment in the issue.